### PR TITLE
Update CoreFx and CoreClr from rc3-24128-00 to rc3-24208-04

### PR DESCRIPTION
**Pull request by zhili1208**

### CODE REVIEW

**Comment by zhili1208** [comment]
@rrelyea

**Comment by zhili1208** [comment]
https://github.com/NuGet/Home/issues/2929

**Comment by emgarten** [comment]
:shipit:

File: test/NuGet.Core.Tests/NuGet.Frameworks.Test/project.json [hunk 1]
```diff
@@ -19,7 +19,7 @@
       "dependencies": {
         "moq.netcore": "4.4.0-beta8",
         "dotnet-test-xunit": "1.0.0-rc2-173361-36",
-        "System.Diagnostics.TraceSource": "4.0.0-rc3-24128-00",
+        "System.Diagnostics.TraceSource": "4.0.0-rc3-24208-04",
         "Microsoft.NETCore.App": {
           "type": "platform",
           "version": "1.0.0-rc3-004338"
```
Comment (eerhardt) [inline test/NuGet.Core.Tests/NuGet.Frameworks.Test/project.json:25]: Don't you also want the corresponding `Microsoft.NETCore.App` version?

That should be `1.0.0-rc3-004425`.

**Pull request merged**